wfl/Database/MongoDBManager.cs

```diff
@@ -26,6 +27,16 @@ public MongoDBManager()
             await collection.InsertOneAsync(r);
         }
 
+        public async Task<RestaurantDto> UpdateRestaurantNameAsync(string id, string name)
```

Comment:
So back to my question from the other file...when restaraunt adds additional fields, what will happen to this class / method?